File: src/3232.find-if-digit-game-can-be-won.3515/solution.ts
function canAliceWin (nums: number[]): boolean {
    let one = 0;
    let two = 0;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] < 10) {
            one += nums[i];
        } else {
            two += nums[i];
        }
    }
    return one !== two;
}

File: src/3232.find-if-digit-game-can-be-won.3515/solution.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int a = 0;
        int b = 0;
        for(int n : nums){
            if(n<10){
                a += n;
            }else{
                b += n;
            }
        }
        return a != b;
    }
}